src/KvkClient.php

```diff
@@ -39,6 +39,21 @@ public function getProfile(QueryInterface $profileQuery): KvkPaginatorInterface
         return $this->profileResponseFactory->fromProfileData($data);
     }
 
+    /**
+     * preform search query
+     * @param QueryInterface $profileQuery
+     * @return KvkPaginatorInterface
+     * @throws KvkApiException
+     * @author Patrick Development <info@patrickdevelopment.nl>
+     */
+    public function getSearch(QueryInterface $profileQuery): KvkPaginatorInterface
```

Comment:
I called it this way because it is consistence with getProfile function with is also get external data. I think it remove get and only call it search than

Reply:
I understand. There are quite a few things that are not up to our current standards. However, that doesn't mean we shouldn't aim to do better with newer code, although I do understand that, given the examples, it is impossible for contributors to guess what we consider "better". 
Please don't take it personally, we do appreciate your contribution. :)

In the case of `getProfile` it is actually getting something concrete, as opposed to the code you built, which just gets "something". In the case of `getProfile` we would use `fetchProfile` instead, because it is getting something specific but from a remote location. This differentiation helps us understand immediately if there is a remote call involved or not. :)